### PR TITLE
add testcase test_zaak_archiefactiedatum

### PR DESCRIPTION
**Pull request by annashamray**
**Changes:**
1. add new testcase "test_zaak_archiefactiedatum.py" for testing calculation of `zaak.archiefactiedatum` based on `resultaat.archiefactietermijn`
1. add fixture for `ResultaatType` model in ZTC
1. upgrade docker image versions of ZTC and ZRC

### CODE REVIEW

File: tests/test_zaak_archiefactiedatum.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+"""
+Test archive procedure of Zaak:
+purpose: to test calculation of Zaak.archiefactiedatum
+In this testcase we check case when Zaak.archiefactiedatum = brondatum of Zaak + ResultaatType.archiefactietermijn
+"""
+import pytest
+
+from .constants import (
+    CATALOGUS_UUID, RESULTAATTYPE_UUID, ZAAKTYPE_UUID, STATUSTYPE_2_UUID
```
Comment (sergei-maertens) [inline tests/test_zaak_archiefactiedatum.py]: isort

**Review by sergei-maertens — APPROVED** [review]

**Comment by sergei-maertens** [comment]
test this please

**Comment by sergei-maertens** [comment]
add to whitelist

File: tests/test_zaak_archiefactiedatum.py (hunk 1 above)
Comment (annashamray) [reply tests/test_zaak_archiefactiedatum.py]: fixed

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**